tests/unit/dojson/test_dojson_institutions.py

```diff
@@ -447,25 +447,22 @@ def test_field_activity_from_372__a():
     assert expected == result['field_activity']
 
 
-def test_name_variants_from_410__a_9():
+def test_name_variants_from_invalid_source_from_marcxml_410_9():
     snippet = (
+        '<record>'
```

Comment:
When you only have a single `<datafield>` you don't need to wrap it in a `<record>`. You can remove this.